### ISSUE TITLE
Add object for Secondary Receivers

### ISSUE THREAD

**Issue by marceloverdijk** [restfb/restfb#807]
The get secondary receivers endpoint (https://developers.facebook.com/docs/messenger-platform/reference/handover-protocol/secondary-receivers) return a list of receivers identified by `id` and `name` pairs.

Can we add an object for that so we can easily parse the response with RestFB?

**Comment by nbartels** [comment]
What about the `NamedFacebookType`? This type contains `name` and `id`. Okay, `metadata` and `type` are part of the object, too. But do we need a special type?

**Comment by marceloverdijk** [comment]
Yes we can use `NamedFcebookType` then, but like you said it contains more, not applicable to the receivers..

**Comment by nbartels** [comment]
I just checked the response and it is a list of `Application` types. In the Facebook documentation only name and id is mentioned, but calling the endpoint without and fields parameter returns in my case `id`, `name` and `link`. I used `me/secondary_receivers?fields=id,name,link,daily_active_users` for example and this is working, too.

So I think the Facebook docs are insufficient and we should add the information to the RestFB docs.
